Source/Scene/ClippingPlaneCollection.js

```diff
@@ -502,8 +497,8 @@ define([
         }
 
         // Use of Plane objects will be deprecated.
-        // But until then, we have no way of telling if they changed since last frame, so we have to do a full udpate.
-        var refreshFullTexture = this._multipleDirtyPlanes || this._containsUntrackablePlanes;
+        // But until then, we have no way of telling if they changed since last frame, so we have to do a full update.
+        var refreshFullTexture = this._multipleDirtyPlanes;
```

Comment:
When more than one `ClippingPlane` is updated in between rendered frames, we refresh the entire clipping plane texture instead of just selectively updating a few adjacent pixels.
Looks like with the new changes though we can replace all uses of `refreshFullTexture` below with just direct access to `this._multipleDirtyPlanes`.